content/en/developers/marketplace/offering.md

```diff
@@ -88,7 +91,9 @@ REST API Integrations send the following types of data to Datadog:
 - [Incidents][10]
 - [Security Events][11]
 
-A Datadog API key is required to submit data to a Datadog API endpoint, and an application key is required to query data from Datadog or create resources on the Datadog site. Optionally, you can setup [OAuth for an integration][15] in a Marketplace tile instead.
+A Datadog API key is required to submit data to a Datadog API endpoint, and an application key is required to query data from Datadog or create resources on the Datadog site. Datadog recommends using [OAuth for an integration][15] to securely authorize third-party application access on a Marketplace tile.
```

Comment:
```suggestion
A Datadog API key is required to submit data to a Datadog API endpoint, and an application key is required to query data from Datadog or create resources on the Datadog site. Instead of requesting these credentials directly from a user, Datadog recommends using [OAuth][15] to handle authorization and access for API-based integrations.
```